examples/detectron2-retrain/mypackage/utility.py

```diff
@@ -13,6 +14,24 @@ def create_new_version_dir():
     shutil.copytree(latest_version_dir, new_version_dir)
     return new_version_dir
 
+
+def auto_compare_and_register(model, eval_metric, model_name, lower, p_metric):
+    client = mlflow.tracking.MlflowClient()
```

Comment:
Kyknya lebih baik dijadiin parameter aja mlflow client-nya.
```suggestion
def auto_compare_and_register(client, model, eval_metric, model_name, lower, p_metric):
```

Reply:
Noted.